src/components/annotation-form.jsx

```diff
@@ -17,79 +18,101 @@ import {Version} from '../services/web-monitoring-db';
  * @param {AnnotationFormProps} props
  */
 export default class AnnotationForm extends React.Component {
-    constructor (props) {
-        super(props);
-        this._onFieldChange = this._onFieldChange.bind(this);
+  constructor(props) {
+    super(props);
+    this._onFieldChange = this._onFieldChange.bind(this);
+    this._onNotesChange = this._onNotesChange.bind(this);
+  }
+
+  render() {
+    const annotation = this.props.annotation || {};
+    const common = {
+      formValues: annotation,
+      onChange: this._onFieldChange
+    };
+
+    const classes = ['annotation-form'];
+    if (this.props.collapsed) {
+      classes.push('annotation-form--collapsed');
     }
 
-    render () {
-        const common = {
-            formValues: this.props.annotation,
-            onChange: this._onFieldChange
-        };
+    return (
+      <form className={classes.join(' ')}>
+        <div className="annotation-form__signifiers">
+          <div className="signifier-container">
+            <h5>Individual Page Changes</h5>
+            <ul className="signifier-list">
```

Comment:
I'm using a list because the `li` elements are helping to contain and style the checkbox controls. See comment below about why they're kept separate.

Did change it to `<ol>`